src/Gandi.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Nexylan packages.
+ *
+ * (c) Nexylan SAS <contact@nexylan.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nexy\Gandi;
+
+use fXmlRpc\Client as FxmlrpcClient;
+use fXmlRpc\Proxy;
+use fXmlRpc\Transport\HttpAdapterTransport;
+use Http\Adapter\Guzzle6\Client as GuzzleClient;
+use Http\Client\HttpClient;
+use Http\Discovery\HttpClientDiscovery;
+use Http\Message\MessageFactory\GuzzleMessageFactory;
+use Nexy\Gandi\Api\AbstractApi;
+
+/**
+ * @author Jérôme Pogeant <p-jerome@hotmail.fr>
+ */
+final class Gandi
+{
+    /**
+     * @var HttpClient
+     */
+    private $httpClient;
+
+    /**
+     * @var string
+     */
+    private $apiUrl;
+
+    /**
+     * @var string
+     */
+    private $apiKey;
+
+    /**
+     * @param HttpClient $httpClient
+     * @param string     $apiUrl
+     * @param string     $apiKey
+     */
+    public function __construct(HttpClient $httpClient, string $apiUrl, string $apiKey)
```

Comment:
Add a default for apiUrl with production url.

Example: https://github.com/nexylan/nexycrypt/blob/master/src/NexyCrypt.php#L29